commit 6bb606e03f9bb3314bc9979f40adadd6f1a222f0 (origin/wasapi_capture_leak)
Author: Matthew Gregan <kinetik@flim.org>
Date:   Thu Nov 3 17:13:59 2016 +1300

    wasapi: Fix leak of stm->capture_client in close_wasapi_stream.

diff --git a/src/cubeb_wasapi.cpp b/src/cubeb_wasapi.cpp
index cc970c6..042f92d 100644
--- a/src/cubeb_wasapi.cpp
+++ b/src/cubeb_wasapi.cpp
@@ -1748,11 +1748,14 @@ void close_wasapi_stream(cubeb_stream * stm)
   SafeRelease(stm->output_client);
   stm->output_client = NULL;
   SafeRelease(stm->input_client);
-  stm->capture_client = NULL;
+  stm->input_client = NULL;
 
   SafeRelease(stm->render_client);
   stm->render_client = NULL;
 
+  SafeRelease(stm->capture_client);
+  stm->capture_client = NULL;
+
   SafeRelease(stm->audio_stream_volume);
   stm->audio_stream_volume = NULL;
 
